sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -591,3 +591,49 @@
   font-family: "BCSans-bold", "BCSans", "Noto Sans", Verdana, Arial, sans-serif;
   color: $brand-blue;
 }
+
+.restrictions-details {
```

Comment:
mm.. we do have a class already for extra-small 
![image](https://user-images.githubusercontent.com/77353155/197045237-1347cd88-ab87-4415-b8f4-0d06c3ad4121.png)
for your case, you can use `value-extra-small`

Reply:
ok. Thanks!

Reply:
please remove this, i beleive its no longer used

Reply:
ok. Removed it. Thanks.